admin-dev/themes/new-theme/js/pages/order/create/customer-manager.js

```diff
@@ -146,7 +146,7 @@ export default class CustomerManager {
    * @private
    */
   _search(searchPhrase) {
-    if (searchPhrase.length < 3) {
+    if (searchPhrase.length === 0) {
```

Comment:
I guess it's WIP debug settings and will be revert for final PR ? 😄

Reply:
The initial search phrase now searches from first input, but the purpose of this exact line was to disable search if like the whole input text is deleted as in legacy page. (not sure if its really needed tho  :thinking: )